modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/saml/SAMLInvalidIssuerTestCase.java

```diff
@@ -67,34 +61,33 @@
     private static final String ATTRIBUTE_CS_INDEX_VALUE = "1239245949";
     private static final String ATTRIBUTE_CS_INDEX_NAME = "attrConsumServiceIndex";
 
-    // User Attributes
-    private static final String USERNAME = "testUser";
-    private static final String PASSWORD = "testUser";
-    private static final String EMAIL = "testUser@wso2.com";
-    private static final String NICKNAME = "testUserNick";
-
     private static final String ACS_URL = "http://localhost:8490/travelocity.com/home.jsp";
     private static final String COMMON_AUTH_URL = "https://localhost:9853/commonauth";
     private static final String SAML_SSO_LOGIN_URL =
             "http://localhost:8490/travelocity.com/samlsso?SAML2.HTTPBinding=HTTP-Redirect";
     private static final String SAML_ERROR_NOTIFICATION_PATH = "/authenticationendpoint/samlsso_notification.do";
 
-    //Claim Uris
-    private static final String firstNameClaimURI = "http://wso2.org/claims/givenname";
-    private static final String lastNameClaimURI = "http://wso2.org/claims/lastname";
-    private static final String emailClaimURI = "http://wso2.org/claims/emailaddress";
-
-    private static final String profileName = "default";
-
     private ApplicationManagementServiceClient applicationManagementServiceClient;
     private RemoteUserStoreManagerServiceClient remoteUSMServiceClient;
     private DefaultHttpClient httpClient;
 
     private boolean isSAMLReturned;
 
+    private SAMLConfig config;
+
+    @Factory(dataProvider = "samlConfigProvider")
+    public SAMLInvalidIssuerTestCase(SAMLConfig config) {
+
+        if (log.isDebugEnabled()){
+            log.info("SAML SSO Test initialized for " + config);
```

Comment:
change to `log.debug`

Reply:
shouldn't it be `config.toString() `?

Reply:
Fixed https://github.com/wso2/product-is/pull/9477/commits/70d73b570425b27e4ff199a8c93aec0c7fb47645